velox/exec/Driver.cpp

```diff
@@ -1090,6 +981,72 @@ folly::dynamic Driver::toJson() const {
   return obj;
 }
 
+template <typename Func>
+auto Driver::withDeltaCpuWallTimer(
+    Operator* op,
+    TimingMemberPtr timingMember,
```

Comment:
why not just pass CpuWallTiming& here? Thanks!

Reply:
My consideration was to hide the implementation details and minimize the code size. For example, in the different call sites like `op->stats().wlock()->finishTiming.add(elapsedSelfTime)` and `op->stats().wlock()->getOutputTiming.add(elapsedSelfTime)`, the `op->stats().wlock()` is common, and the user of the timer should NOT need to worry about how to access the stats and what type of lock it needs to take. The user just need to know what kind of timing it is, whether it's finishTiming, or getOutputTiming. Moreover, if you directly pass in `op->stats().wlock()->finishTiming`, the code would become longer.